docs/repo-docs/getting-started/add-to-existing-repository.mdx

```diff
@@ -179,13 +179,27 @@ Add `.turbo` to your `.gitignore` file. The `turbo` CLI uses these folders for p
 
 </Step>
 <Step>
-### Run the `build` and `check-types` tasks with `turbo`
+### Add a `packageManager` field to root `package.json`
+
+Turborepo optimizes your repository using information from your package manager. To declare which package manager you're using, add a [`packageManager`](https://nodejs.org/api/packages.html#packagemanager) field to your root `package.json` if you don't have one already.
```

Comment:
Should we advertise [--dangerously-disable-package-manager-check](https://turbo.build/repo/docs/reference/run#--dangerously-disable-package-manager-check) here? I don't want to encourage people to use it, but if for some reason this is a blocker for people we should try to help them unblock themselves.

Reply:
Yes.